.goreleaser.yml

```diff
@@ -51,4 +51,4 @@ release:
   # Visit your project's GitHub Releases page to publish this release.
   draft: true
```

Comment:
If the test I described in the PR description is successful, I will follow up with a change to set `draft: false` so that releases are completely automated.

Since our CI job is very extensive, I don't see a reason not to have the entire process be automated. Let me know if you think otherwise.